doc/releases/release-notes-3.3.rst

```diff
@@ -2592,3 +2592,999 @@ Known Issues
+* :github:`54236` - b_u585i_iot02a_ns: Can't build TFM enabled samples (TF-M 1.7.0)
+* :github:`54230` - STM32H7: Kernel crash with BCM4=0
+* :github:`54225` - Intel CAVS: tests/lib/c_lib/ fails
+* :github:`54224` - Issues with picolibc on xtensa platforms
+* :github:`54223` - Intel CAVS: tests/kernel/common/kernel.common.picolibc fails
+* :github:`54214` - Display framebuffer allocation
+* :github:`54210` - tests: drivers: udc drivers.udc fails
+* :github:`54209` - USB C PD dead battery support
+* :github:`54208` - Various RISC-V FPU context switching issues
+* :github:`54205` - Regression: RiscV FPU regs not saved in multithreaded applications
+* :github:`54202` - decawave_dwm1001_dev: i2c broken due to pinctrl_nrf fix
+* :github:`54190` - RP2040 cannot be compiled with C11 enabled
+* :github:`54173` - Bluetooth: GATT: Change awareness of bonded GATT Client is not maintained on reconnection after reboot
+* :github:`54172` - Bluetooth: GATT: Written value of gatt_cf_cfg data may be dropped on power down
+* :github:`54148` - qemu_x86_tiny places picolibc text outside of pinned.text
+* :github:`54140` - BUS FAULT when running nmap towards echo_async sample
+* :github:`54139` - Bluetooth: Audio: race hazard bt_audio_discover() callback vs unicast_client_ase_cp_discover()
+* :github:`54138` - Buetooth: shell: `bt adv-data` isn't working properly
+* :github:`54136` - Socket error after deregistration causes RD client state machine to re-register
+* :github:`54123` - Intel CAVS 25: tests/boards/intel_adsp/ssp fails
+* :github:`54117` - West flash fails on upload to Nucleo F303K8
+* :github:`54104` - Bluetooth: Host: Bonding information distribution in the non-bondable mode
+* :github:`54087` - tests:igmp:frdm_k64f: igmp test fails
+* :github:`54086` - test:rio:frdm_k64f: rio user_space test fails with zephyr-v3.2.0-3842-g7ffc20082023
+* :github:`54078` - No activity on can_tx when running the ISO-TP sample
+* :github:`54072` - Bluetooth: Host: Periodic scanner does not differentiate between partial and incomplete data
+* :github:`54065` - How To Change C++ Version Compilation Option For Freestanding Application?
+* :github:`54053` - CI:frdm_k64f: kernel.common.stack_protection test failure
+* :github:`54034` - QSPI: Unable to build the project when introduced DMA into external flash interfacing
+* :github:`54017` - Modules: TF-M: Resolve QCBOR issues with TF-M 1.7.0
+* :github:`54005` - esp32 Severe crash using modules with embedded PSRAM (eg esp32-wroom-32E-n8r2)
+* :github:`54002` - mgmt: mcumgr: bluetooth transport: Inability to use refactored transport as a library in some circumstances
+* :github:`53995` - drivers: ethernet: stm32: Enable ethernet statistics in the driver
+* :github:`53994` - net: ethernet: Multicast receive packets statistics are not getting updated
+* :github:`53991` - LE Audio: samples/bluetooth/broadcast_audio_sink configure error
+* :github:`53990` - Duplicate
```

Comment:
I get that this is the legit title of the issue, but is it really something to include here?

Reply:
Removed